automaxlair/MaxLairInstance.py

```diff
@@ -410,9 +410,12 @@ def calculate_ore_cost(self, num_resets: int) -> int:
         """Calculate the prospective Dynite Ore cost of resetting the game."""
         return 0 if num_resets < 3 else min(10, num_resets)
 
-    def check_sufficient_ore(self, num_resets: int) -> bool:
+    def check_sufficient_ore(self, current_resets_count: int, aditionnal_reset_count: int) -> bool:

```

Comment:
Looks great; the last thing is to drop the current reset count argument and just use self.consecutive resets within the function.